lib/101/statefull_life_cycle.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+class StatefullLifeCycleLearn extends StatefulWidget {
+  const StatefullLifeCycleLearn({super.key, required this.message});
+
+  final String message;
+
+  @override
+  State<StatefullLifeCycleLearn> createState() =>
+      _StatefullLifeCycleLearnState();
+}
+
+class _StatefullLifeCycleLearnState extends State<StatefullLifeCycleLearn> {
+  String _message = " ";
```

Comment:
Since you learnt usage of `late`, consider using it while declaring this variable as well. Because empty string (" ") assignment is redundant.

```suggestion
  late String _message;
```

Reply:
Done.